spidy/cam_cap/capture.cpp

```diff
@@ -0,0 +1,29 @@
+#include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <opencv2/highgui/highgui.hpp>
```

Comment:
why camera stuff is in tests PR?

Reply:
i don't know why camera branch commits added to that pull request too ?
i will try to investigate this problem